example-graphs.html

```diff
@@ -0,0 +1,439 @@
+<html lang="en">
+    <head>
+        <title>Example data visualizations</title>
+
+        <style>
+            body {
+                font-family: 'Georgia', 'serif' !important;
+            }
+
+            div.page-container {
+                width: 600px;
+                margin-left: calc(50% - 300px);
+                margin-right: calc(50% - 300px);
+            }
+
+            @media screen and (max-width: 625px) {
+                div.page-container {
+                    margin-left: 10px;
+                    width: calc(100% - 20px);
+                }
+            }
```

Comment:
When the screen hits 625 >=, we then opt for these dimensions. In theory, this allows phone things.